proposals/context.md

```diff
@@ -0,0 +1,265 @@
+# context-protocol
```

Comment:
I think we can go with title-cased names here as the name of the protocol.

Reply:
Changed to `Context Protocol` in bccaeac